jdat_notebooks/contributing.md

```diff
@@ -0,0 +1,245 @@
+# James Webb Space Telescope Data Analysis Tool Notebooks
+
+.. note::
+
+   ``jdat_notebooks`` are one component of STScI's larger `Data Analysis Tools Ecosystem <https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis>`_.
+
+   .. note::
+
+   ``jdat_notebooks`` are intended for the broader science community.  If you wish to contribute notebooks or technical expertise, we refer you to `dat_pyinthesky <https://github.com/spacetelescope/dat_pyinthesky/tree/master/jdat_notebooks>`_.
```

Comment:
```suggestion
   The `jdat_notebooks repository <https://github.com/spacetelescope/jdat_notebooks>`_ is for already-reviewed notebooks that are ready for consumption by the science community.  If you wish to contribute notebooks or technical expertise, we refer you to `dat_pyinthesky <https://github.com/spacetelescope/dat_pyinthesky/tree/master/jdat_notebooks>`_ as the place to submit a first draft.  The description below outlines the whole process, which begins in `dat_pyinthesky` but finishes in `jdat_notebooks`.
```

I was not entirely sure if you meant this point or not, but I'm suggesting it on the theory that this document will be ported over to the jdat_notebooks *repository* as well.  If I was mis-understanding, though, @orifox , feel free to ignroe this suggestion